### PR TITLE
[SDK][Typescript][Python] Add network array to getEscrows filter

### PR DESCRIPTION
**Pull request by flopez7**
## Description

Add network array to getEscrows filter to retrieve information from all networks

## How test the changes

`yarn test` for Typescript and `make run-test` for Python

## Operational checklist

- [x] All new functionality is covered by tests
- [x] Any related documentation has been changed or added

### CODE REVIEW

**Comment by leric7** [comment]
We need to update SDK doc on gitbook as well.

**Review by leric7 — CHANGES_REQUESTED** [review]
We have /examples directory for both Python/Typescript SDK. Let's update example, as well. This way, you can make sure your updates are working end-to-end. In addition, we need to update SDK docs on GitBook.

**Comment by flopez7** [comment]
> We have /examples directory for both Python/Typescript SDK. Let's update example, as well. This way, you can make sure your updates are working end-to-end. In addition, we need to update SDK docs on GitBook.

@leric7 I have it in mind, I was waiting for the approval to update the documentation only once.

**Review by portuu3 — APPROVED** [review]

**Pull request merged**
